legion/CMakeLists.txt

```diff
@@ -61,8 +64,8 @@ endif()
 
 function(add_app_folder folder)
     add_executable("${folder}" "${folder}/main.cpp" "${folder}/taco-generated.cpp")
-    target_include_directories("${folder}" PRIVATE "${folder}")
-    target_link_libraries("${folder}" distal-runtime-lib Legion::Legion openblas)
+    target_include_directories("${folder}" PRIVATE "${folder}" "${TBLIS_INCLUDE_DIRS}")
+    target_link_libraries("${folder}" distal-runtime-lib Legion::Legion openblas "${TBLIS_LIBRARIES}")
```

Comment:
Is there no equivalent of Tblis::Tblis like the legion install? just curious.

Reply:
Just made a TBLIS::TBLIS target. It's possible, just extra work.